mvn-extension/src/test/java/org/arquillian/smart/testing/mvn/ext/dependencies/SurefireApiDependencyTest.java

```diff
@@ -108,6 +108,23 @@ public void should_fail_when_autocorrect_repeats_the_strategy()
 
     }
 
+    @Test
+    public void should_remove_junit5_platform_dependency() {
+        // given
+        Model model = prepareModelWithSurefirePlugin("2.19.1");
+        final Configuration conf = configureWithAutocorrect("nwe");
+        final DependencyResolver dependencyResolver = new DependencyResolver(conf);
+        final Plugin plugin = model.getBuild().getPlugins().get(0);
+        plugin.addDependency(new DependencyResolver.JUnit5SurefireProviderDependency());
```

Comment:
I understand your point but, it is not so easy because in some tests you need both `model` and `dependencyResolver`. One possible solution would be create a method with so many parameters so all possibilities can be fit with this method call.

The other option would be create some classes using builder pattern, but my question is is it worth for just 4 lines when the code is duplicated in just 3 test methods to do this?